performant-software/dm-2
# Color selection for text and canvas highlights

### What does this PR do?
- Adds a color picker element to the top of the viewer frame for each text and canvas document
- Gives new highlights the color selected in the document's color picker
- Changes the backdrop for canvas documents from white to black

### What issues does it address?
- Closes #30 
- Closes #31 
- Closes #32 

### How to test
- At the [review app](https://dm-2-staging-pr-33.herokuapp.com/), click on the 'Dummy Project' link to enter a project view populated with a few non-persistent documents in the grid. These should include two text and two canvas (image) documents. 
- Text documents should include an editor menu with a highlighter icon at the beginning. Select some text and click this icon to create a new highlight. 
- Above the highlighter icon should be a color swatch that, when clicked, opens a popup color picker. Select a new color and create a new highlight, confirming that each highlight is the correct color.
- Canvas documents should include a 'Rectangle' button (a stand-in for an actual set of drawing tools) that adds a new draggable/resizable highlight shape on top of the image. Above it, the same swatch should appear. Repeat the above with a canvas resource, creating a few highlights of different colors.
